bucket/irfanview.json

```
@@ -1,11 +1,16 @@
 {
     "version": "4.60",
-    "description": "A fast, compact and innovative graphic viewer",
-    "homepage": "https://www.irfanview.com",
+    "description": "A fast, compact and innovative FREEWARE (for non-commercial use) graphic viewer (with PlugIns)",
+    "homepage": "https://www.irfanview.com/",
```

```suggestion
    "description": "A fast, compact and innovative graphic viewer (with PlugIns)",
    "homepage": "https://www.irfanview.com",
```

Same reason as above